src/main/java/com/nyanbot/DukeGuiControllers/DukeMainWindow.java

```diff
@@ -0,0 +1,73 @@
+package com.nyanbot.DukeGuiControllers;
```

Comment:
Should this be in all lower case, as per [coding standard](https://se-education.org/guides/conventions/java/intermediate.html) for naming convention for packages? I noticed the same in several other places too.